tests/contest/runtimetest/src/tests.rs

```diff
@@ -546,6 +546,27 @@ pub fn test_io_priority_class(spec: &Spec, io_priority_class: IOPriorityClass) {
     }
 }
 
+pub fn test_validate_root_readonly(spec: &Spec) {
+    let root = spec.root().as_ref().unwrap();
+    if root.readonly().unwrap() {
+        if let Err(e) = test_dir_write_access("/") {
+            let errno = Errno::from_raw(e.raw_os_error().unwrap());
+            if errno == Errno::EROFS {
+                /* This is expected */
+            } else {
+                eprintln!("readonly root filesystem, error in testing write access for path /");
+            }
+        }
```

Comment:
This is ok, but also check for read access is `Ok` and not err _if possible_. For the `!= EROFS` case, also add the actual `errno` in the `eprintl` for debugging

Reply:
I fixed this.
https://github.com/youki-dev/youki/pull/2976/commits/c3c62fbf865aeaedfb6fdce1b33aba31301988ca